doc/storagemond/storagemond-hld.md

```diff
@@ -382,6 +383,7 @@ disk_io_writes          = STRING                    ; Describes the total number
 reserved_blocks         = STRING                    ; Describes the reserved blocks count of the SSD                                         (Dynamic)
 firmware                = STRING                    ; Describes the Firmware version of the SSD                                              (Dynamic)
 health                  = STRING                    ; Describes the overall health of the SSD as a % value based on several SMART attrs      (Dynamic)
+last_sync_time          = STRING                    ; The latest successful sync time of disk attribtes to STATE_DB in UNIX timestamp format (Dynamic)
```

Comment:
@assrinivasan can you update the time stamp format to MM:DD:YYYY format instead of epoch time which is not user convenient

Reply:
Done in latest